projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss

```diff
@@ -1,25 +1,20 @@
 @import "../shared/styles/host-styles.scss";
 
 :host {
-
-
   --wvr-dropdown-menu-animation-speed: 0.25s;
   --wvr-dropdown-menu-background: var(--wvr-white);
   --wvr-dropdown-menu-border: 1px solid var(--wvr-gray-dark);
-  --wvr-dropdown-menu-border-radius: 6px;
+  --wvr-dropdown-menu-border-radius: 0.375rem; //6px;
```

Comment:
Is there a reason we are using rem as opposed to px?

Reply:
Using px instead of rem.